views/pages/add-profile.ejs

```diff
@@ -0,0 +1,16 @@
+<%- include('../partials/head'); %>
+<%- include('../partials/header'); %>
+<h1>Profiel aanmaken</h1>
+<form action=/add method=post enctype=multipart/form-data> <label>
+  Foto
+  <input name=foto type=file accept=image/*> </label> <label>Naam<input name=naam></label>
+  <label>
+    leeftijd
+    <input name=leeftijd>
+  </label>
+  <label>
+    <textarea name=bio rows=5></textarea>
+  </label>
+  <button>Add</button>
+</form>
```

Comment:
I'm curious to see how we add images with the use of Multer! But we do need an input to add these here.